rpc/rpc-wrappers.go

```diff
@@ -1016,6 +1020,30 @@ func NewStorageAuthorityServer(rpc RPCServer, impl core.StorageAuthority) error
 		return
 	})
 
+	rpc.Handle(MethodCountCertificatesRange, func(req []byte) (response []byte, err error) {
+		var countReq struct {
+			Start time.Time
+			End   time.Time
+		}
+
+		err = json.Unmarshal(req, &countReq)
+		if err != nil {
+			return
+		}
+
+		count, err := impl.CountCertificatesRange(countReq.Start, countReq.End)
+		if err != nil {
+			return
+		}
+		buf := new(bytes.Buffer)
+		err = binary.Write(buf, binary.LittleEndian, count)
```

Comment:
In various places where only a single value is returned (or sent as a argument) we use binary encoding seemingly to save the unnecessary JSON bytes (e.g. `AlreadyDeniedCSR`, `AddCertificate`, both `GetCertificate...` methods, etc).